lang/lang_en_fr.txt

```diff
@@ -946,51 +938,51 @@
 "Resuming print"
 "Reprise de l'impr."
 
-#MSG_BED_CORRECTION_RIGHT c=14 r=1
+#MSG_BED_CORRECTION_RIGHT c=14
 "Right side[um]"
 "Droite [um]"
 
-#MSG_RPI_PORT
+#MSG_RPI_PORT c=13
 "RPi port"
 "Port RPi"
 
 #MSG_WIZARD_RERUN c=20 r=7
 "Running Wizard will delete current calibration results and start from the beginning. Continue?"
 "Lancement de l'Assistant supprimera les resultats actuels de calibration et commencera du debut. Continuer?"
 
-#MSG_SD_CARD
+#MSG_SD_CARD c=8
 "SD card"
 "Carte SD"
 
-#MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY
+#MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY c=8
 "FlashAir"
 "FlshAir"
 
-#
+#MSG_RIGHT c=10
 "Right"
 "Droite"
 
-#MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=60
+#MSG_FIND_BED_OFFSET_AND_SKEW_LINE1 c=20 r=2
 "Searching bed calibration point"
 "Recherche du point de calibration du plateau chauffant"
```

Comment:
```suggestion
"Recherche point calibration du plateau"
```
In other places, we call the build plate `plateau`, instead of `plateau chauffant`, we can do the same here. Additionally, we can remove the first `du ` and `de ` without completely loosing the meaning of the sentence, however it now sounds more like a command to look up calibration points for the bed.

Reply:
Still and issue because of text warp, see
```
[E]: Text is longer than definition on line 965: cols=20 rows=2 (rows diff=1)
 source text:
   1 |Searching bed       |
   2 |calibration point   |
 translated text:
   1 |Recherche point     |
   2 |calibration du      |
   3 |plateau             |
```

Reply:
Is `Recherche point ca- libration du plateau` allowed? It would look like this
```
  -01234567890123456789-
1 |Recherche point ca- |
2 |libration du plateau|

Reply:
As `ça` is a french word and with the dash, especially with the new line after it, I would read that something like "Research point to this: liberation(with a spelling mistake) of the plane(as in flat expanse, not plane, flying transportation)" 

Maybe we can use `Je cherche point du calibrage` ("Searching for calibration point") or `Je cherche point du calibrage plateau` ("Searching for the calibration point for the bed"), although neither of those option are as clear as `Recherche du point de calibration du plateau chauffant`.